api/openapi.yaml

```diff
@@ -1633,6 +1633,8 @@ components:
         launchTime:
           type: string
           format: date-time
+        architecture:
+          type: string
```

Comment:
Should this be an enum so that it can be standardised among cloud providers? I'm aware of at least 3 different ways things represent x64 it can be: x86_64 or x64 or AMD64.

I would prefer that we pick one and make that the standard for the API and UI this will be especially important when we start mixing multiple cloud providers reporting assets and you want to use an odata filter to find all the x64 VMs for example.